pkg/controller/kubefedcluster/controller.go

```diff
@@ -184,7 +184,9 @@ func (cc *ClusterController) addToClusterSet(obj *fedv1b1.KubeFedCluster) {
 	// create the restclient of cluster
 	restClient, err := NewClusterClientSet(obj, cc.client, cc.fedNamespace, cc.clusterHealthCheckConfig.Timeout)
 	if err != nil || restClient == nil {
-		cc.RecordError(obj, "MalformedClusterConfig", errors.Wrap(err, "The configuration for this cluster may be malformed"))
+		cc.RecordError(obj, "MalformedClusterConfig", errors.Wrap(err, "The configuration for this "+
+			"cluster may be malformed"))
+		utilruntime.HandleError(errors.Wrapf(err, "the configuration for this cluster may be malformed"))
```

Comment:
Why don't you use klog ?

Reply:
We are not using utilruntime anywhere else in the code.

Reply:
here we used it
https://github.com/kubernetes-sigs/kubefed/blob/4859cebef7f11e61277a76b61e1f6a8189fcee63/pkg/controller/kubefedcluster/controller.go#L195
if you think using utilruntime is not necessary, I will change it to klog.